### PR TITLE
Automatic replacement of verb

### PR DESCRIPTION
**Pull request by redfast00**
See  #278

### CODE REVIEW

File: style-guide.tex [hunk 1]
```diff
@@ -97,7 +97,7 @@ \section{Displaying example code}
 \section{Displaying inline code}
 \label{sec:displaying-inline-code}
 
-To display inline code inside the document, you can use the \verb|\verb| command.
+To display inline code inside the document, you can use the \lstinline|\lstinline| command.
 This should be used when talking about Linux commands, filenames, \ldots
 When you want inline code inside a title, you can use the \verb|\texttt{}| command.
 When using \verb|\texttt|, you will need to escape certain characters.
```
Comment (boegel) [inline style-guide.tex]: Mention why not to use `\verb`?

**Comment by redfast00** [comment]
Checking that `\verb` isn't reintroduced is blocked by #283

File: run-checks.sh [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
 
+result=$(find . -type f -name "*.tex" | xargs -n1 ./find_white_in_terminal.py)
 if [[ $result ]]; then
     echo "Found one or more LaTeX commands in prompt environment, please close them with {}"
     echo "$result"
     ec=1
 fi
 
+# Style guide can use \verb to say it's not okay to use \verb
+result=$(grep -R --include=\*.tex '\\verb[^\w]' | grep -v style-guide)
```
Comment (boegel) [inline run-checks.sh]: @redfast00 Usage of `grep` is `grep [OPTIONS] PATTERN [FILE...]`, and `[^\w]` doesn't do what you think it does (I think)?

**Comment by redfast00** [comment]
@boegel I've fixed it and tested it.

**Pull request merged**